packages/openid4vc-issuer/src/OpenId4VcIssuerApi.ts

```diff
@@ -0,0 +1,27 @@
+import type { IssueCredentialOptions, SendCredentialOfferOptions } from './OpenId4VcIssuerServiceOptions'
```

Comment:
Might be good to include this in a separate PR. Would not want to confuse the users as the functionality is not here, yet.